htpasswd.go

```diff
@@ -42,15 +44,23 @@ import (
 // CheckInterval represents how often we check for changes in htpasswd file.
 const CheckInterval = 30 * time.Second
 
+const passwordCacheDuration = time.Minute
```

Comment:
Perhaps make this public so that this constant appears in godoc? `PasswordCacheDuration`

Reply:
I've added a short comment and made the constant public